src/datasets/workflows/process_nsclc_sc_zuani/config.vsh.yaml

```
@@ -0,0 +1,86 @@
+name: process_nsclc_sc_zuani
+namespace: datasets/workflows
+
+argument_groups:
```

Can we add a section for the input file?

```suggestion
argument_groups:
  - name: Inputs
    arguments:
      - type: file
        name: --input
        description: Path to the dataset
        required: true
        example: "https://ftp.ebi.ac.uk/biostudies/fire/E-MTAB-/526/E-MTAB-13526/Files/10X_Lung_Tumour_Annotated_v2.h5ad"